vscode/src/ruby/chruby.ts

```diff
@@ -99,46 +138,51 @@ export class Chruby extends VersionManager {
   private async discoverRubyVersion(): Promise<RubyVersion> {
     let uri = this.bundleUri;
     const root = path.parse(uri.fsPath).root;
+    let version: string;
```

Comment:
I noticed that we were unintentionally catching our custom error for invalid `.ruby-version` files on the same catch as the one intended for non-existing files. 

That's incorrect. When we find a `.ruby-version` file, but the contents are invalid, we want to raise and not continue going up the directories.